### PR TITLE
[misc] Various cleanup

### PR DESCRIPTION
**Pull request by Djailla**
- Adding in AUTHORS file
- Fix minimal Django version in setup
- Remove deprecated options in travis config

### CODE REVIEW

**Comment by yakky** [comment]
This has been integrated as part of #452

**Comment by Djailla** [comment]
@yakky nope. This is a new PR

File: tox.ini [hunk 1]
```diff
@@ -1,9 +1,9 @@
 [tox]
 envlist =
 	py35-django{21,22}
-	py36-django{21,22,30,31}
-	py37-django{21,22,30,31}
-	py38-django{21,22,30,31}
+	py36-django{21,22,30,31,master}
+	py37-django{21,22,30,31,master}
+	py38-django{21,22,30,31,master}
 	docs
```
Comment (yakky) [inline tox.ini:7]: this has been removed on purpose from declared environments to allow to run `tox` without any failure

File: AUTHORS.rst [hunk 2]
```diff
@@ -19,6 +19,7 @@ Contributors
 * Andrew Dodd
 * Angel Velasquez
 * Austin Matsick
+* Bastien Vallet (Django 3.1 compat)
```
Comment (yakky) [inline AUTHORS.rst:22]: remove `(Django 3.1 compat)` section, this files only tracks contributors names

**Comment by chrisglass** [comment]
Thanks!

**Pull request merged**